src/main/resources/db/hsqldb/schema.sql

```diff
@@ -20,14 +28,52 @@ DROP TABLE products IF EXISTS;
 
 -- Create table to hold information about the products
 CREATE TABLE products (
-    id      INTEGER IDENTITY PRIMARY KEY,
-    sku     VARCHAR(30) NOT NULL UNIQUE,
-    category_id INTEGER NOT NULL,
-    collection_id INTEGER NOT NULL,
-    name    VARCHAR(30),
-    price   DECIMAL(20,2),
-    description VARCHAR(500),
-    photo VARCHAR(500),
-    FOREIGN KEY (category_id) REFERENCES categories(id),
-    FOREIGN KEY (collection_id) REFERENCES collections(id)
+  id              INTEGER IDENTITY PRIMARY KEY,
```

Comment:
Is the indenting correct here?

Reply:
@jgwilson42 I've seen that style in a few examples but I've also seen no indenting. 

I can change it if you want?

Reply:
I would also prefer reducing the indenting tbh.